test/test_joint_trajectory_splitter.py

```diff
@@ -0,0 +1,305 @@
+#! /usr/bin/env python
+
+import rospy
+import actionlib
+import control_msgs.msg
+import roslaunch
+import rospkg
+import trajectory_msgs.msg
+import pytest
+import copy
+from giskardpy import logging
+
+@pytest.fixture(scope=u'function')
+def init_ros():
+    rospy.init_node('JointTrajectorySplitterTest')
```

Comment:
This should have a module scope. It's unnecessary to init a ros node for every test.